scipy/fft/__init__.py

```diff
@@ -44,6 +44,15 @@
    dstn - N-D Discrete sine transform
    idstn - N-D Inverse discrete sine transform
 
+Fast Hankel Transforms
+======================
+
+.. autosummary::
+    :toctree: generated/
+
+    fht - Fast Hankel transform
+    ifht - Inverse of fht
```

Comment:
Locally this makes `python runtests.py -n --refguide-check --submodule fft` pass for me:
```suggestion
   fht - Fast Hankel transform
   ifht - Inverse of fht
```